Packs/Gmail/Integrations/Gmail/Gmail.py

```diff
@@ -191,6 +191,45 @@ def parse_mail_parts(parts):
     return body, html, attachments
 
 
+def format_fields_argument(fields: list[str]) -> list[str] | None:
```

Comment:
the unit test uses string-type `fields`, please make sure the type hints are correct.